sw/device/lib/dif/dif_edn.h

```diff
@@ -0,0 +1,384 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_EDN_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_EDN_H_
+
+/**
+ * @file
+ * @brief <a href="/hw/ip/edn/doc/">Entropy Distribution Network</a> Device
+ * Interface Functions
+ *
+ * This API implements the interface for the Entropy Distribution Network (EDN)
```

Comment:
Should we make a point of eventualy(tm) writing up file comments for each DIF? (probably worth filing a tracking bug)

Reply:
Filed lowrisc/opentitan#6242